packages/components/src/Icon/Icon.component.js

```diff
@@ -36,12 +36,29 @@ export const TRANSFORMS = Object.keys(FA_TRANSFORMS);
  * @example
 <Icon name="fa-bars"></Icon>
  */
-function Icon({ className, name, title, transform, onClick }) {
+function Icon({ className, name, title, transform, onClick, href }) {
```

Comment:
Please name the image source to `src` rather than `href` to comply with the HTML tag attribute name.